### PR TITLE
[EVW-1727] Add 'applying for someone else' page

### PR DESCRIPTION
**Pull request by roc**
### adds 'Applying for someone else' page
- [x] pages
- [x] fields
- [x] steps
- [x] validation 
- [x] acceptance tests

### multi-toggle/dependent issue

I've added an issue (#27) around multiple dependent fields and multiple toggle
This behaviour isn't currently handled by hof, because it's something of an eccentric design pattern. We'll need to deal with in the validation pass

applying for someone else

### CODE REVIEW

File: apps/apply/fields/applicant-contact-details.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  'applicant-contact-details': {
+    className: ['form-group']
+  }
+};
```
Comment (shepda) [inline apps/apply/fields/applicant-contact-details.js]: Shouldn't this be a required field?

Comment (roc) [inline apps/apply/fields/applicant-contact-details.js]: Yes it should! Have fixed in a rebase now

File: apps/apply/fields/applicant-email-address.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+module.exports = {
+  'applicant-email-address': {
+    validate: ['required'],
+    dependent: {
+      field: 'applying-for-minor',
+      value: 'yes'
+    }
+  }
+};
```
Comment (roc) [inline apps/apply/fields/applicant-email-address.js]: You're absolutely right, but that's not supported in hof at the moment. I've added an issue #27 to cover that, but it'll take some fundamental changes that — I think — are out of scope for this PR

Comment (shepda) [inline apps/apply/fields/applicant-email-address.js]: ok

**Pull request merged**